### PR TITLE
Bgrd dev

### PR DESCRIPTION
**Pull request by BGRD-dev**
single array used

### CODE REVIEW

File: Countries/ViewController.swift [hunk 1]
```diff
@@ -39,18 +37,20 @@ class ViewController: UIViewController {
...
-                    for arrayData in countriesArray
-                    {
-                        let country = arrayData as! [String:Any]
-                        guard let name = country["name"]
-                            else { return }
-                        nameArray.append(name as! String)
-                        guard let capital = country["capital"]
+                    for arrayData in countriesArray {
+                        
+                        let countryData = arrayData as! [String:Any]
```
Comment (antivixen) [inline Countries/ViewController.swift:42]: more extra variables?

File: Countries/ViewController.swift [hunk 2]
```diff
@@ -39,18 +37,20 @@ class ViewController: UIViewController {
...
+                        guard let name = countryData["name"],
+                            let capital = countryData["capital"],
+                            let imageName = countryData["imageName"]
                             else{ return }
-                        capitalArray.append(capital as! String)
-                        guard let imageName = country["imageName"]
-                            else{ return }
-                        imageNameArray.append(imageName as! String)
+                        
+                        let country = Country(name: name as? String ?? "",
```
Comment (antivixen) [inline Countries/ViewController.swift]: any chance the ```name``` is gonna not to be a ```String```?

File: Countries/CountryClass.swift [hunk 3]
```diff
@@ -20,5 +20,3 @@ class Country {
         self.imageName = imageName
     }
 }
-
-
```
Comment (Egregor2011) [inline Countries/CountryClass.swift:24]: nits

**Review by antivixen — CHANGES_REQUESTED** [review]
lgtm, few minor things though

**Pull request merged**